docs/jdbc-mysql-catalog.md

```diff
@@ -100,9 +100,27 @@ Please refer to [Manage Metadata Using Gravitino](./manage-metadata-using-gravit
 MySQL doesn't support Gravitino `Boolean` `Fixed` `Struct` `List` `Map` `Timestamp_tz` `IntervalDay` `IntervalYear` `Union` `UUID` type.
 :::
 
+#### Table column auto-increment
+
+- Supports setting auto-increment.
+:::notice
+MySQL setting an auto-increment column requires simultaneously setting a unique index; otherwise, an error will occur.
+:::
+
 ### Table properties
 
-- Doesn't support table properties.
+- Supports setting table engine property. The default value is `InnoDB`. You can set the table engine property to `MyISAM` 、 `MEMORY`、 `CSV`、 `ARCHIVE`、 `BLACKHOLE`、 `FEDERATED`、 `ndbinfo`、 `MRG_MYISAM`、 `PERFORMANCE_SCHEMA`.
+- Supports setting table auto-increment-offset property. Used to specify the starting value of the auto-increment field.
```

Comment:
Can you use a table to list all the supported table properties like other documentation.